conf/script/build-ha-io

```diff
@@ -517,6 +527,16 @@ motr_kernel_rsc_add() {
                                            kind=Optional symmetrical=false
 }
 
+var_motr_rsc_add() {
+    log "${FUNCNAME[0]}: Adding var-motr mounts to Pacemaker..."
+    sudo pcs -f $cib_file resource create var-motr1 ocf:heartbeat:Filesystem \
+        device=$lvolume directory=/var/motr1 \
```

Comment:
@andriytk , now the /var/motr device will be mounted on /var/motr1 and /var/motr2 respectively, do we need to update /etc/sysconfig/m0d-* env files accordingly? as the local motr env file has /var/motr while the guest has /var/motr1 or /var/motr2 as per the node.

Reply:
I thought about it. We can do this - yes. But I decided to leave it with /var/motr symlink for now for the sake of less changes...

Reply:
Okay, also when the node is replaced the directories  won't be present. We need to recreate the directories and symlinks as part of the restore operation.

Reply:
Yes, which function is called on the restore?

Reply:
The functions that need to be invoked for restore need to add "restore" to them as in "bootstrap_update_restore".

Reply:
Updated the patch. Please verify. Thanks!

Reply:
Strange, I updated the branch - (see https://github.com/andriytk/cortx-ha/tree/var-motr), but it is not showing here.

Reply:
not able to see the change yet

Reply:
This is weird. Let me create another PR...